src/Util/Serializer/SerializerInstance.php

```diff
@@ -29,18 +29,30 @@ private function __construct()
     public static function getInstance(): Serializer
     {
         if (is_null(self::$instance)) {
-            self::$instance = SerializerBuilder::create()->
-            setPropertyNamingStrategy(
-                new SerializedNameAnnotationStrategy(
-                    new IdenticalPropertyNamingStrategy()
+            self::$instance = SerializerBuilder::create()
+                ->setPropertyNamingStrategy(
+                    new SerializedNameAnnotationStrategy(
+                        new IdenticalPropertyNamingStrategy()
+                    )
                 )
-            )->
-            configureHandlers(function (HandlerRegistry $registry) {
-                $registry->registerHandler(self::DIRECTION['deserialization'], 'MoySklad\Entity\MetaEntity', 'json', new MetaEntityDeserializeHandler());
-                $registry->registerHandler(self::DIRECTION['deserialization'], 'MoySklad\Entity\Barcode', 'json', new BarcodeDeserializeHandler());
-            })->
-            addDefaultHandlers()->
-            build();
+                ->configureHandlers(
+                    function (HandlerRegistry $registry) {
+                        $registry->registerHandler(
+                            self::DIRECTION['deserialization'],
+                            'MoySklad\Entity\MetaEntity',
```

Comment:
```suggestion
                            MoySklad\Entity\MetaEntity::class,
```
Может лучше так? Одно дело, в аннотациях никуда не деться от строк с классами, а в коде ведь можно этого избежать